charts/dapr/charts/dapr_dashboard/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Dapr Dashboard
 name: dapr_dashboard
-version: 0.7.0
+version: 1.1.0-rc.1
```

Comment:
@artursouza is there a new version of dashboard???

Reply:
@mukundansundar This is for the version of the chart, not the image. @artursouza and I discussed offline.

Reply:
@pkedy I was referring to the CRDs. For dashboard, the version is different. Can we keep the chart version in sync with the tag version?

Reply:
@artursouza I think unfortunately the chart version (0.7.0) has surpassed the tag version (0.6.0) so we are out of sync on this one. Thoughts?

Reply:
My thought it to revert the chart back to 0.7.0 since nothing has changed and figure this out later :)

Reply:
Let's keep dashboard separate for now and keep others in sync.